src/components/LeftSidebar/ConversationsList/Conversation.vue

```diff
@@ -163,6 +163,30 @@
 						{{ t('spreed', 'Notify about calls') }}
 					</NcActionButton>
 				</template>
+
+				<template v-if="supportImportantConversations || supportSensitiveConversations">
+					<NcActionSeparator />
+
+					<NcActionButton v-if="supportImportantConversations"
+						type="checkbox"
+						:model-value="item.isImportant"
+						@click="toggleImportant(!item.isImportant)">
+						<template #icon>
+							<IconMessageAlert :size="16" />
+						</template>
+						{{ t('spreed', 'Important conversation') }}
```

Comment:
Unpopular opinion, should we add the description here as well?

Reply:
I'd say no, as these are shortcuts? And this description is what you're supposed to see once

Reply:
But this might/is the first place where users might see it?